docs/resources/aws_nat_gateway.md

```diff
@@ -0,0 +1,117 @@
+---
+title: About the aws_nat_gateway Resource
+---
+
+# aws\_nat\_gateway
+
+Use the `aws_nat_gateway` InSpec audit resource to test the properties of a single AWS NAT gateway.
+
+## Syntax
+
+An `aws_nat_gateway` resource block declares the tests for a single AWS NAT gateway by id, name, vpc_id or subnet_id.
+
+    describe aws_nat_gateway(id: 'nat-abc0123456789deff') do
+      it { should exist }
+    end 
+
+    describe aws_nat_gateway(name: 'my-nat-gateway') do
+      it { should exist }
+    end
+
+Multiple parameters can be provided for better granularity.
+
+    describe aws_nat_gateway(vpc_id: 'vpc-abc01234', subnet_id: 'subnet-6789deff') do
+      it { should exist }
+    end
+
+
+#### Parameters
+
+At least one of the following parameters must be provided.
+- id
+- name
+- subnet_id
+- vpc_id
+
+##### id _(required if none of the others provided)_
```

Comment:
Perhaps instead of adding `_(required if none of the others provided)_` to each param, you could add
`_*At least one of the below required*_`
Or if you prefer having it with each param, perhaps re-word slightly `_(required if no other parameter supplied)_`